vignettes/articles/startup_messages.Rmd

```diff
@@ -0,0 +1,149 @@
+---
+title: "How to suppress package startup messages"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{How to suppress package startup messages}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+When making a reprex, you might have to load libraries that are producing some messages and warnings. The goal of this article is to suppress these, without suppressing all messages and warnings of the reprex, as they might be useful for debugging. This article aims at answering the issue [#187](https://github.com/tidyverse/reprex/issues/187). 
+
+We'll review the different possibilities to silence messages and warnings when loading a library using using a fake package that gives messages and warnings upon loading.
+
```

Comment:
I think you should show the techniques used right here, in one or more `eval = FALSE` chunks, for the reader who just wants to quickly see a couple of ideas and not read a longer article.

See the chunk near the top of this readxl vignette that starts "For now, here are a few ways `read_excel()` can look when you take control of the geometry:" for inspiration:

https://readxl.tidyverse.org/articles/sheet-geometry.html

Oftentimes, people just want a quick reminder of syntax they've seen before but can't remember.